src/rollup/plugins/externals.ts

```diff
@@ -77,7 +78,7 @@ export function externals (opts: NodeExternalsOptions): Plugin {
       if (opts.trace === false) {
         return {
           ...resolved,
-          id: normalizeid(resolved.id),
+          id: isAbsolute(resolved.id) ? normalizeid(resolved.id) : resolved.id,
```

Comment:
Here we could also have explicit check for possible npm import name before normalizing (or it could be also handled from mlly.normalizeid to not wrongly add file protocol for them)

Reply:
I checked normalizeid but its description says that it is meant to always add a protocol.

Reply:
This really should never ever be hit, but if it _is_ hit it is guaranteed to be a bug. A non-absolute path at this point will become `file://someid` which is an invalid URL and cannot be imported. So I think the check is right.

Reply:
It still doesn't makes sense to me why we add a condition for something is not supposed to happen and hiding the issue.. Anyway not worth discussing returning as is for non absolute is okay.